lxd/container_lxc.go

```diff
@@ -1339,22 +1339,16 @@ func (c *containerLXC) initLXC(config bool) error {
 	for _, k := range c.expandedDevices.DeviceNames() {
 		m := c.expandedDevices[k]
 		if shared.StringInSlice(m["type"], []string{"unix-char", "unix-block"}) {
-			// Prepare all the paths
-			srcPath, exist := m["source"]
+			// destination paths
+			destPath, exist := m["path"]
 			if !exist {
-				srcPath = m["path"]
+				destPath = m["source"]
 			}
```

Comment:
This reads a bit weird to me as path is a required property of unix-char/unix-block, so it will always exist.

Reply:
I don't think so, not according to our docs and not how we handle it internally. :)